datasets/us/ofac/press_releases/crawler.py

```diff
@@ -0,0 +1,240 @@
+from lxml.html import HtmlElement, fromstring, tostring
+from pydantic import BaseModel, Field
+from rigour.mime.types import HTML
+from typing import List, Literal
+
+from zavod import Context
+from zavod import helpers as h
+from zavod.shed.gpt import DEFAULT_MODEL, run_typed_text_prompt
+from zavod.stateful.review import (
+    Review,
+    assert_all_accepted,
+    request_review,
+    get_review,
+    model_hash,
+    html_to_text_hash,
+)
+
+# Rough conversion: 1 GPT token ≈ 4 chars English text
+TOKEN_CHAR_RATIO = 4
+MAX_TOKENS = 3000
+MAX_CHARS = MAX_TOKENS * TOKEN_CHAR_RATIO
```

Comment:
is it still too big and requires splitting we prompt with text_content() instead of the html?